templates/defineOperation.dot

```diff
@@ -0,0 +1,23 @@
+{-| {{= it.doc }}
+{{? it.input && it.input.members }}
+__Parameters__
+
+{{~ it.input.members :m }}* `{{= m.key }}` __:__ `{{= m.value.type }}`
+{{~}}
+{{?}}
+-}
+{{= it.name }} : {{? it.input }}{{~ it.input.members :m }}{{= m.value.type }} -> {{~}}{{?}}AWS.ServiceConfig -> AWS.Http.UnsignedRequest {{= it.output.type }}
+{{= it.name }} {{? it.input }}{{~ it.input.members :m }}{{= m.key }} {{~}}{{?}}serviceConfig =
+    AWS.Http.unsignedRequest
```

Comment:
Operations return an `UnsignedRequest a` instead of an `Http.Request`. The user then has to sign the request using `AWS.Signers.V4.sign`, passing in a `Date` along with the unsigned request.

Getting the current `Date` requires an effect, so it can't be done automatically for the user